client/src/Game.cpp

```diff
@@ -118,15 +107,27 @@ sf::View Game::calculateViewport() {
 }
 
 void Game::handlePacket(const AresProtocol::AresMessage &message) {
-    switch (message.message_case()){
+    switch (message.message_case()) {
         case AresProtocol::AresMessage::kModifyObject:
             handleMsgModifyObject(message.modifyobject());
             break;
         default:
             break;
     }
 }
-
 void Game::handleMsgModifyObject(const AresProtocol::ModifyObject &modifyObject) {
-    //To be filled
+    if (modifyObject.action_case() == modifyObject.kCreate) {
```

Comment:
The value should accessed with `AresProtocol::ModifyObject::kCreate` 

Moreover, you should do a `switch` instead. It should be clearer.